### PR TITLE
Blog - Manage dotfiles from anywhere with Git

### PR DESCRIPTION
**Pull request by msali123**
Done with all checks.

### CODE REVIEW

File: .github/styles/HouseStyle/tech-terms/general.txt [hunk 1]
```diff
@@ -206,3 +206,10 @@ instrumenta
 proport
 uncommented
 keyphrase
+
+dotfiles
+dotfile
+bashrc
+rf
+hardlink
```
Comment (adamgordonbell) [inline .github/styles/HouseStyle/tech-terms/general.txt:214]: should be two words, according to google, hard link.

Comment (msali123) [reply .github/styles/HouseStyle/tech-terms/general.txt:214]: Should I amend here too? @adamgordonbell 
image

Comment (msali123) [reply .github/styles/HouseStyle/tech-terms/general.txt]: Hey Adam should I?

Comment (msali123) [reply .github/styles/HouseStyle/tech-terms/general.txt]: Hey Adam should I?

Comment (adamgordonbell) [reply .github/styles/HouseStyle/tech-terms/general.txt]: Hey @msali123 ,
If it's inside of a script block or in `ticks` then you don't need to add it to the keyword list, generally.

If you have it only in scripts and in ticks and it still complains that you'll have to add it. When it complains about something that is in ticks, I try to add it to the `nonsense` file to indicate that it shouldn't need to be in the list but yet I had to put it there.

**Comment by msali123** [comment]
@adamgordonbell got it

Pushed to review.

**Comment by msali123** [comment]
@jalletto

**Pull request merged**